src/ActionsImporter/Services/DockerService.cs

```diff
@@ -45,6 +47,15 @@ public Task ExecuteCommandAsync(string image, string server, string version, par
             actionsImporterArguments.Add(dockerArgs);
         }
 
+        // Forward the current user's UID/GID to the container
+        // to ensure the output files are owned by the current user
+        if (_runtimeService.IsLinux)
+        {
+            var userId = _processService.RunAndCaptureSync("id", "-u");
+            var groupId = _processService.RunAndCaptureSync("id", "-g");
```

Comment:
Any reason not to use the existing `RunAndCaptureAsync` version here?

```suggestion
            var userId = await _processService.RunAndCaptureAsync("id", "-u");
            var groupId = await _processService.RunAndCaptureAsync("id", "-g");
```

Reply:
This method isn't `async`, so it cannot await the result.

It currently (a little ways down) returns `return _processService.RunAsync(...)`.

There could be something I'm misunderstanding though!

Reply:
Ah, I didn't notice that! This method isn't async, but it already returns a `Task` so it should be easy to update it by adding the `async` keyword to line 36:

```csharp
public async Task ExecuteCommandAsync(string image, string server, string version, params string[] arguments)
```

Then line 63 can await the result before returning:

```csharp
return await _processService.RunAsync(/*...*/);
```Ah, I didn't notice that! This method isn't async, but it already returns a `Task` so it should be easy to update it by adding the `async` keyword to line 36:

```csharp
public async Task ExecuteCommandAsync(string image, string server, string version, params string[] arguments)
```

Then line 63 can await the result before returning:

```csharp
return await _processService.RunAsync(/*...*/);
```

Reply:
Perfect, thanks!